include/alpaka/block/shared/dyn/BlockSharedMemDynMember.hpp

```diff
@@ -0,0 +1,119 @@
+/* Copyright 2020 Jeffrey Kelling
+ *
+ * This file is part of Alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#include <alpaka/block/shared/dyn/Traits.hpp>
+#include <alpaka/core/Assert.hpp>
+
+#include <type_traits>
+#include <array>
+
+#ifndef ALPAKA_BLOCK_SHARED_DYN_MEMBER_ALLOC_KB
+#define ALPAKA_BLOCK_SHARED_DYN_MEMBER_ALLOC_KB 30
```

Comment:
kibi-byte or kilo-byte?

Reply:
Kibibyte (1024). I guess there is a wrong comment about this somewhere, I will check.

Reply:
May we want to call it `TStaticAllocKiB` and `ALPAKA_BLOCK_SHARED_DYN_MEMBER_ALLOC_KIB`. Hoewever, the macro version does not make it easier...

Reply:
The capital 'K' already makes it not be the SI 'kilo', but then, it could also be capitalized only because of camel-case in the template param.